superset-frontend/src/chart/ChartRenderer.jsx

```diff
@@ -59,6 +60,11 @@ const defaultProps = {
   triggerRender: false,
 };
 
+const isDevMode = process.env.WEBPACK_MODE === 'development';
+if (isDevMode) {
+  setConfig({ logLevel: 'debug', trackTailUpdates: false });
+}
```

Comment:
This does not change what users saw, but does avoid rendering the chart two times at hot reload.  See: https://github.com/gaearon/react-hot-loader#out-of-bound-warning